release-notes/10.0/preview/preview3/api-diff/Microsoft.NETCore.App/10.0-preview3_System.ComponentModel.Annotations.md

```diff
@@ -0,0 +1,35 @@
+# System.ComponentModel.Annotations
+
+```diff
+  namespace System.ComponentModel.DataAnnotations
+  {
+      public sealed class ValidationContext : System.IServiceProvider
+      {
+-         [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("The Type of instance cannot be statically discovered and the Type's properties can be trimmed.")]
++         [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Constructing a ValidationContext without a display name is not trim-safe because it uses reflection to discover the type of the instance being validated in order to resolve the DisplayNameAttribute when a display name is not provided.")]
```

Comment:
@tarekgh do we want this attribute excluded, or do you prefer it shows in the diff?

Reply:
The change is only the on the text and not on the APIs, so it is ok to exclude the changes in the attributes here.

We need only to show the following in the diff

```C#
public ValidationContext(object instance, string displayName, System.IServiceProvider? serviceProvider, System.Collections.Generic.IDictionary<object, object?>? items);
```

Reply:
Okay, so I fixed it in the tool in my latest commit, but maybe we can leave this specific one since the text is quite larger.